make/Makefile.plugin.mk

```diff
@@ -20,7 +20,7 @@ clean-plugin:
 
 ## build-plugin: Builds the plugin.
 build-plugin:
-	cd ${PLUGIN_DIR} && yarn install && yarn build
+	cd ${PLUGIN_DIR} && yarn install --network-timeout 3600000 && yarn build
```

Comment:
Should we add this to the dockerfile as well? https://github.com/kiali/openshift-servicemesh-plugin/blob/main/plugin/Dockerfile#L8